_news/167.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: post
+id: 167
+title: "Erlang/OTP 27.0 Release Candidate 1"
+lead: "Erlang/OTP 27.0-rc1 is the first release candidate for OTP 27"
+tags: "release, OTP, 27.0-rc1, Release Candidate"
+date: "2024-02-14"
```

Comment:
If you temporarily set this date to todays date you will be able to see the rendered text here: https://deploy-preview-121--erlang-org.netlify.app/news/167